app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -159,6 +155,10 @@ class StoryFragmentTest {
   @Test
   fun testStoryFragment_correctStoryCountLoadedInHeader() {
     launch<StoryActivity>(createStoryActivityIntent()).use {
+      storyProgressTestHelper.markPartialStoryProgressForFractions(
```

Comment:
I suggest creating a helper function, might help in future as well to prevent the repetition of this code, and call it before launching the screen as it is the data modification which we do before the screen gets launched. 

```kotlin
private fun setStoryPartialProgress() {}
```

```kotlin
@Test
  fun testStoryFragment_correctStoryCountLoadedInHeader() {
    setStoryPartialProgress
    launch<StoryActivity>(createStoryActivityIntent()).use {....}
    ....
}
```

for other tests too.